packages/fx-core/templates/plugins/resource/function/bicep/function.template.bicep

```diff
@@ -0,0 +1,122 @@
+param functionServerfarmsName string
+param functionAppName string
+@minLength(3)
+@maxLength(24)
+@description('Name of Storage Accounts for function backend.')
+param functionStorageName string
+param aadClientId string
+@secure()
+param aadClientSecret string
+param m365TenantId string
+param applicationIdUri string
+param oauthAuthorityHost string
+
+{{#contains 'fx-resource-frontend-hosting' Plugins}}
+param frontendHostingStorageEndpoint string
+{{/contains}}
+{{#contains 'fx-resource-azure-sql' Plugins}}
+param sqlDatabaseName string
+param sqlEndpoint string
+{{/contains}}
+{{#contains 'fx-resource-identity' Plugins}}
+param identityId string
+{{/contains}}
+
+var teamsMobileOrDesktopAppClientId = '1fec8e78-bce4-4aaf-ab1b-5451cc387264'
+var teamsWebAppClientId = '5e3ce6c0-2b1f-4285-8d4b-75ee78787346'
+var authorizedClientApplicationIds = '${teamsMobileOrDesktopAppClientId};${teamsWebAppClientId}'
+
+resource functionServerfarms 'Microsoft.Web/serverfarms@2020-06-01' = {
+  name: functionServerfarmsName
+  location: resourceGroup().location
+  sku: {
+    name: 'Y1'
+  }
+  kind: 'functionapp'
+  properties: {
+  }
+}
+
+resource functionApp 'Microsoft.Web/sites@2020-06-01' = {
+  kind: 'functionapp'
+  name: functionAppName
+  location: resourceGroup().location
+  properties: {
+    reserved: false
+    serverFarmId: functionServerfarms.id
+    siteConfig: {
+      cors: {
+        allowedOrigins: [
+          frontendHostingStorageEndpoint
+        ]
+      }
+      numberOfWorkers: 1
+    }
+  }
+}
+
+resource functionStorage 'Microsoft.Storage/storageAccounts@2021-04-01' = {
+  kind: 'StorageV2'
```

Comment:
for `name`, `kind`, `location`, we'd better align the sequence of them for all the resources.

Reply:
sure